### PR TITLE
Indicate severity in the suggestor list of messages

### LINKED ISSUE #4916: Add table view to UI suggester to indicate severity
Propose to replace the TextEdit widget with something that can display items in a table manner.
We can colorize the background based on the severity of the item.

E.g. red for exceptions, and yellow background for warnings.

Alternatively icons can be added left side of the text.

Related to adjustments proposed here: https://github.com/equinor/ert/issues/4811

### PR DESCRIPTION
**Pull request by DanSava**
**Issue**
Resolves #4916 

**Approach**

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Updated documentation
- [x] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by DanSava** [comment]
Messages that need to be displayed to the user in the suggestion dialog will look like this:

Screenshot from 2023-02-21 09-04-58

Please suggest (pun intended) a new colour for the suggestion messages if purple is not royal enough :smile_cat: .

File: src/ert/gui/ertwidgets/suggestor_message.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+class SuggestorMessage(QWidget):
+    def __init__(self, lbl_text, color, text):
+        QWidget.__init__(self)
+        common_style = """
+        border-style: outset;
+        border-width: 2px;
+        border-radius: 10px;
+        border-color: beige;
```
Comment (andreas-el) [inline src/ert/gui/ertwidgets/suggestor_message.py:14]: I just wanted to comment on this. The outset isn't really visible when you blend white and beige on the border.
It's no biggie, but maybe not the outset effect you want. I've only tested on windows 11, and rhel 11, but ubuntu seem to show similar results also.
If you replace the `beige` with `darkgrey` you see what i mean.

File: src/ert/gui/ertwidgets/suggestor_message.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        return SuggestorMessage("ERROR", color, text)
+
+    @classmethod
+    def warning_msg(cls, text):
+        color = "#ff8000"
+        return SuggestorMessage("WARNING", color, text)
+
+    @classmethod
+    def suggestion_msg(cls, text):
+        color = "#9535ab"
```
Comment (andreas-el) [inline src/ert/gui/ertwidgets/suggestor_message.py:51]: Personally i'd pick light blue or something more friendly, but the royal purple would suffice i guess.

**Review by andreas-el — COMMENTED** [review]
Looks really nice! Had some comments to read at least.

File: src/ert/gui/ertwidgets/suggestor_message.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        )
+
+        wrapper = textwrap.TextWrapper(width=75)
+        self.lbl = QLabel(wrapper.fill(text))
+        self.lbl.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        self.lbl.setStyleSheet(common_style + "background-color: lightgray;")
+
+        self.hbox = QHBoxLayout()
+        self.hbox.addWidget(self.type_lbl)
+        self.hbox.addWidget(self.lbl)
```
[the remark this answers is not in the document]
Comment (andreas-el) [reply src/ert/gui/ertwidgets/suggestor_message.py:36]: But then again, maybe it'll look weird when the messages get different width.

File: src/ert/gui/ertwidgets/suggestor_message.py (hunk 2 above)
Comment (DanSava) [reply src/ert/gui/ertwidgets/suggestor_message.py:51]: :blue_heart:

File: src/ert/gui/ertwidgets/suggestor_message.py (hunk 1 above)
Comment (DanSava) [reply src/ert/gui/ertwidgets/suggestor_message.py:14]: `darkgrey` makes it pop :sparkling_heart:

File: src/ert/gui/ertwidgets/suggestor_message.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (DanSava) [reply src/ert/gui/ertwidgets/suggestor_message.py:36]: With the stretch in place:

Screenshot from 2023-02-22 10-31-51

without the stretch:
Screenshot from 2023-02-22 10-32-59

Personally, I like the one without a stretch in place

[the remark this answers is not in the document]
Comment (andreas-el) [reply src/ert/gui/ertwidgets/suggestor_message.py:34]: Yep. I agree. My vote is on the original behaviour, without stretch in place.

**Pull request merged**